src/library/scala/collection/IndexedSeq.scala

```diff
@@ -88,6 +88,9 @@ trait IndexedSeqOps[+A, +CC[_], +C] extends Any with SeqOps[A, CC, C] { self =>
 
   override def slice(from: Int, until: Int): C = fromSpecific(new IndexedSeqView.Slice(this, from, until))
 
+  override def sliding(size: Int, step: Int): Iterator[C] =
+    new IndexedSeqSlidingIterator[A, CC, C](this, size, step)
+
```

Comment:
You need argument checking here to maintain exception compatibility.  Both size > 0 and step > 0.  The exceptions are of the form `java.lang.IllegalArgumentException: requirement failed: size=0 and step=1, but both must be positive`